core/assets/bundles/bundle_de.properties

```diff
@@ -866,26 +866,26 @@ unit.chaos-array.name = Chaos Array
-tutorial.drill = Manuelles Abbauen von Ressourcen ist ineffizient.\n[accent]Bohrer[] können automatisch abbauen.\n Platziere einen auf einem Kupfer Vorkommen.
-tutorial.drill.mobile = Mining manually is inefficient.\n[accent]Drills []can mine automatically.\nTap the drill tab in the bottom right.\nSelect the[accent] mechanical drill[].\nPlace it on a copper vein by tapping, then press the[accent] checkmark[] below to confirm your selection.\nPress the[accent] X button[] to cancel placement.
-tutorial.blockinfo = Each block has different stats. Each drill can only mine certain ores.\nTo check a block's info and stats,[accent] tap the "?" button while selecting it in the build menu.[]\n\n[accent]Access the Mechanical Drill's stats now.[]
-tutorial.conveyor = [accent]Transportbänder[] werden dazu benutzt Materialien zum Kern zu transportieren.\n Erstelle eine Reihe von Transportbändern zum Kern.
-tutorial.conveyor.mobile = [accent]Conveyors[] are used to transport items to the core.\nMake a line of conveyors from the drill to the core.\n[accent] Place in a line by holding down your finger for a few seconds[] and dragging in a direction.\n\n[accent]{0}/{1} conveyors placed in line\n[accent]0/1 items delivered
+tutorial.next = [lightgray]<Tippen um fortzufahren>
+tutorial.intro = Du befindest dich im[scarlet] Mindustry-Tutorial.[]\nBeginne, indem du[accent] Kupfer abbaust[]. Tippe dazu auf ein Kupfervorkommen in der Nähe deiner Basis.\n\n[accent]{0}/{1} Kupfer
+tutorial.drill = Manuelles Abbauen von Ressourcen ist ineffizient.\n[accent]Bohrer []können automatisch abbauen.\nPlatziere einen Bohrer auf einem Kupfervorkommen.
+tutorial.drill.mobile = Manuelles Abbauen ist ineffizient.\n[accent]Bohrer []können automatsich abbauen.\nTippe auf den Bohrer Tab unten rechts.\nWähle den[accent] Mechanischen Bohrer[].\nPlatziere ihn durch Tippen auf ein Kupfervorkommen, dann klicke auf das[accent] Häkchen[] unten um deine Auswahl zu bestätigen.\nKlicke auf den [accent] X-Buttton[] um den Bau abzubrechen.
+tutorial.blockinfo = Jeder Block hat unterschiedliche Eigenschaften. Bohrer können immer nur eine Art von Material abbauen.\nFür Infos und Eigenschaften eines Blocks wähle einen Block im Baumenü aus und [accent] klicke auf den "?"-Button.[]\n\n[accent]Schau dir jetzt die Eigenschaften des Mechanischen Bohrers an.[]
+tutorial.conveyor = [accent]Förderbänder[] werden dazu benutzt Materialien zum Kern zu transportieren.\nErstelle eine Reihe von Förderbändern zum Kern.
+tutorial.conveyor.mobile = [accent]Förderbänder[] werden dazu benutzt Materialien zum Kern zu transportieren.\nErstelle eine Reihe von Förderbändern zum Kern.\n[accent] Zur Erstellung einer Reihe lasse deinen Finger für einige Sekunden gedrückt[] und ziehe ihn in eine beliebige Richtung.\n\n[accent]{0}/{1} Förderbänder in einer Reihe erstellt\n[accent]0/1 Ressourcen transportiert.
```

Comment:
>Zur Erstellung einer Reihe lasse deinen Finger für einige Sekunden gedrückt[] und ziehe ihn in eine beliebige Richtung.

I find this sentence difficult to understand, might need rewording. Can't think of a better word than "Reihe" though.